lib/charms/tls_certificates_interface/v3/tls_certificates.py

```diff
@@ -1463,15 +1537,16 @@ def __init__(
         self,
         charm: CharmBase,
         relationship_name: str,
-        expiry_notification_time: int = 168,
+        expiry_notification_time: Optional[int] = None,
     ):
         """Generate/use private key and observes relation changed event.
 
         Args:
             charm: Charm object
             relationship_name: Juju relation name
-            expiry_notification_time (int): Time difference between now and expiry (in hours).
+            expiry_notification_time (int): Number of hours prior to certificate expiry.
                 Used to trigger the CertificateExpiring event. Default: 7 days.
+                This value is used as a recommendation only.
```

Comment:
We may want to provide more information as to how this is a recommendation and what users should expect here.